src/components/Empty/Empty.types.ts

```diff
@@ -0,0 +1,40 @@
+import { OcBaseProps } from '../OcBase';
+
+export enum EmptyMode {
+    data = 'data',
```

Comment:
do we maintain a standard approach of capitalizing enum keys?

Reply:
No, the other contributor's favor lowercase, but the components I initially added use the .NET pattern of sentence casing. Those should probably be updated to map to the majority.

Reply:
Ack